packages/vanilla-i18n/src/ContentTranslator.tsx

```diff
@@ -14,6 +14,7 @@ export interface IContentTranslatorProps {
     title: string;
     sourceLocale?: string | null;
     activeLocale?: string | null; // for organizecategories
+    renderNavigationData?: () => void;
```

Comment:
I'd encourage you to think more about the naming of props like this.

This is a component interface for a **translation** package. It has nothing to do with navigation and no knowledge of it.

Something like `afterUpdate` or `afterSave` would be more appropriate.